Apps/WebClient/src/ClientApp/app/views/timeline.vue

```diff
@@ -399,7 +399,7 @@ export default class TimelineComponent extends Vue {
 
   private get numberOfPages(): number {
     let result = Math.ceil(
-      this.filteredEntriesLength / this.getNumberOfEntriesPerPage
+      this.filteredTimelineEntries.length / this.numberOfEntriesPerPage
     );
     if (result < 1) {

```

Comment:
I'd rather see:
let result = 1;
if (this.filteredEntriesLength > this.numberOfEntriesPerPage)
{
     result = Math.ceil(this.filteredTimelineEntries.length / this.numberOfEntriesPerPage);
}    

which is more explicit then the if afterwards.